samples/SimpleWebSampleV2/Controllers/ConfigurationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright 2017 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace SimpleWebSampleV2.Controllers
```

Comment:
Although it's useful having the sample as documentation, I'm wary of adding samples for all of the various configuration options (it'll become a bit unwieldy to maintain). Perhaps we remove this, but look to add a few explanatory lines in the README instead?